server/data/data-api/src/main/java/org/apache/james/rrt/lib/Mapping.java

```diff
@@ -73,6 +75,7 @@ static IdentityMappingPolicy selectIdentityPolicy(Type type) {
             case Error:
             case Group:
             case Address:
+            case Alias:
```

Comment:
It's maybe the time to put this information in `Type`.

Reply:
While I do agree with you I remember I defended a similar position against @mbaechler and I did not end up being the one the most convincing.

Anyway could we consider this being a nice improvement, and create a backlog task for discussing then performing such a refactoring?

Reply:
Typically something like `leaking implementation concerns in API`

Though I agree with you as I prefer your proposal. Can we wait him coming back?

Reply:
@blackheaven : can you answer @chibenwa on this please and see if we can finish that PR or not? Thanks !

Reply:
Yep I did some archeology & did not find the comment I reference. Let's forget about my remark and write @blackheaven proposal.